src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImportsFromStaticMethodRefJavadocDisabled.java

```diff
@@ -0,0 +1,85 @@
+/*
+UnusedImports
+processJavadoc = false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Objects;
+import java.util.List;
+import java.util.Map;
+import java.util.Collections;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.Arrays; // violation 'Unused import - java.util.Arrays.'
+import static java.lang.Integer.parseInt; // violation 'Unused import - java.lang.Integer.'
+import static java.lang.String.format; // violation 'Unused import - java.lang.String.format.'
+import static java.util.List.of; // violation 'Unused import - java.util.List.of.'
+import static java.util.Collections.emptyMap; // violation 'Unused import - java.util.Collections.emptyMap.'
+
+/**
+ * Use {@link Arrays::sort} in this javadoc.
```

Comment:
@mahfouz72 if you haven't already, please take time to read through the discussion at https://github.com/checkstyle/checkstyle/pull/12933#discussion_r1171237410. I completely forgot about this detail, and these test inputs are full of these misleading comments. Please do something like this for all of them:
```suggestion
 * This {@link Arrays::sort} is NOT a valid link, and this check does not count this as a usage.
```

Reply:
@nrmancuso Yes I read it but I thought that we have to put this alert in the input files that have ` processJavdoc = true ` only. But now all is updated

Reply:
> I thought that we have to put this alert in the input files that have processJavdoc = true only

If you think this is confusing, maybe it will be to others also. Let’s consider: in all cases, to the Javadoc tool, this is never a valid link, so we should not consider it a usage of an import. However, if you think we should update the test inputs that have `processJavdoc = false` to have something like `This {@link Arrays::sort} is NOT a valid link.` to make our intention more clear, we can do this.